src/API/Trace/ValidationSpanContext.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\API\Trace;
+
+use function strlen;
+use function strtolower;
+
+class ValidationSpanContext
```

Comment:
I think `SpanContextValidator` is a better name

Reply:
Updated